packages/loader/src/Texture2DLoader.ts

```diff
@@ -71,4 +79,6 @@ export interface Texture2DParams {
   filterMode: TextureFilterMode;
   /** Anisotropic level for texture. */
   anisoLevel: number;
+  /**  Whether to use sRGB color space, default is true. */
```

Comment:
/** Whether to use sRGB color space or linear color space. @defaultValue `true` */